src/Aspire.ProjectTemplates/templates/aspire-empty/.template.config/ide.host.json

```diff
@@ -5,10 +5,7 @@
   "disableHttpsSymbol": "NoHttps",
   "symbolInfo": [
     {
-      "id": "AspireVersion",
-      "isVisible": true,
```

Comment:
Just as FYI, in the aspire-apphost template we left isVisible true but removed here. Is that intentional?

Reply:
Yea this seems to be in error.  Without this set to true, this won’t display on UI